README.md

```diff
@@ -1,4 +1,750 @@
+* Go version 1.14 or above.
+
+## Installation
+The current version of this SDK: 0.0.1
+
+There are a few different ways to download and install the Eventstreams Go SDK project for use by your
+Go application:
+
+#### `go get` command  
+Use this command to download and install the SDK to allow your Go application to
+use it:
+
+```
+go get -u github.ibm.com/IBM/eventstreams-go-sdk 
+```
+
+#### Go modules  
+If your application is using Go modules, you can add a suitable import to your
+Go application, like this:
+
+```go
+import (
+	"github.ibm.com/IBM/eventstreams-go-sdk/pkg/adminrestv1"
+)
+```
+
+then run `go mod tidy` to download and install the new dependency and update your Go application's
+`go.mod` file.
+
+#### `dep` dependency manager  
+If your application is using the `dep` dependency management tool, you can add a dependency
+to your `Gopkg.toml` file.  Here is an example:
+
+```
+[[constraint]]
+  name = "github.ibm.com/IBM/eventstreams-go-sdk"
+  version = "0.0.1"
+
+```
+
+then run `dep ensure`.
+
+## Using the SDK
+For general SDK usage information, please see [this link](https://github.com/IBM/ibm-cloud-sdk-common/blob/main/README.md)
+
+## Questions
+
+If you are having difficulties using this SDK or have a question about the IBM Cloud services,
+please ask a question at 
+[Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-cloud).
+
+## Issues
+If you encounter an issue with the project, you are welcome to submit a
+[bug report](<github-repo-url>/issues).
+Before that, please search for similar issues. It's possible that someone has already reported the problem.
+
+## Open source @ IBM
+Find more open source projects on the [IBM Github Page](http://ibm.github.io/)
+
+## Contributing
+See [CONTRIBUTING](CONTRIBUTING.md).
+
+## License
+
+This SDK project is released under the Apache 2.0 license.
+The license's full text can be found in [LICENSE](LICENSE).
+
+# Event Streams Administration REST API
+
+This REST API allows users of the
+[IBM Event Streams service](https://cloud.ibm.com/docs/services/EventStreams/index.html)
+to administer
+[Kafka topics](#using-the-rest-api-to-administer-kafka-topics)
+associated with an instance of the service. You can use this API to perform the following
+operations:
+  - [Create a Kafka topic](#creating-a-kafka-topic)
+  - [List Kafka topics](#listing-kafka-topics)
+  - [Get a Kafka topic](#getting-a-kafka-topic)
+  - [Delete a Kafka topic](#deleting-a-kafka-topic)
+  - [Update a Kafka topic configuration](#updating-kafka-topics-configuration)
+  - [List which topics are mirrored](#list-current-mirroring-topic-selection)
+  - [Replace selection of topics which are mirrored](#replace-mirroring-topic-selection)
+  - [List active mirroring topics](#list-active-mirroring-topics)
+  
+The Admin REST API is also [documented using swagger](./admin-rest-api.yaml).
+
+Note: This Admin REST API works with both Enterprise plan and Standard plan.
```

Comment:
Isn't it all plans?

Reply:
I meant what bout Lite? Especially as it's mentioned just below. I don't consider Classic a plan as it does not exist anymore

Reply:
Should we add lite then or just all plans

Reply:
I don't think this sentence is adding any value, so removed,